ex5/server/services/item_manager.js

```diff
@@ -0,0 +1,114 @@
+const PokemonClient = require('../clients/pokemon_client');
+const { Items } = require('../db/models');
+const { v4: uuidv4 } = require('uuid');
+
+class ItemManager {
+	constructor() {
+		this.pokemonClient = new PokemonClient();
+	}
+
+	getItems = async () => await Items.findAll();
+
+	handleItem = async (item) => {
+		if (this._isNumber(item)) {
+			const pokemon = await this.fetchAndAddPokemon(item);
+			return pokemon;
+		}
+		if (this._isList(item)) {
+			return await this.fetchAndAddManyPokemon(item);
+		}
+
+		await this.addItem(item);
+	};
+
+	addItem = async (item) => {
+		const user = await Items.findOne({
+			where: { ItemName: item },
+		});
+		if (!user) {
+			await Items.create({
+				id: uuidv4(),
```

Comment:
id is of type integer - which uuid is not (it's a string)
also - id is auto-populated you don't need to specify it at all
this line crashes all inserts - I had to comment it out